### PR TITLE
Introduces Support for Proton Calendar

### PR DESCRIPTION
**Pull request by jonathansampson**
Care was taken to avoid updating any earlier code (that can be done in a separate PR). Following the merging of this PR, another effort should be made to harmonize the implementations for Google Calendar and Proton Calendar.

This commit does the following:
- Adds support for i18n
- Simplifies Brave Talk SVG Icon
- Updates manifest to support Proton Calendar
- Begins harmonization work in Brave Talk module
- Introduces Common module for general utils, etc.
- Modifies content script to initialize Proton Cal code
- Adds Proton Calendar module

### CODE REVIEW

**Comment by jonathansampson** [comment]
We'll need to make sure the version number gets bumped.

File: src/brave-talk.ts [hunk 1]
```diff
@@ -19,6 +32,48 @@ export function isBraveTalkUrl(text: string): boolean {
...
+  const featureList = features.toLowerCase().split(",");
+
+  // Some features are required, for security/privacy reasons.
+  for (const feature of ["noopener", "noreferrer"]) {
+    if (!featureList.includes(feature)) {
+      featureList.push(feature);
+    }
+  }
+
+  window.open(url, name, featureList.join(","));
```
[the remark this answers is not in the document]
Comment (jonathansampson) [reply src/brave-talk.ts:74]: `noopener` is enforced, as well as `noreferrer`.

**Comment by jonathansampson** [comment]
I just realized no changes were made to the Popup logic; so visiting Proton Calendar will still present "Create Google Calendar Meeting" in the extension popup. I'll take a look at this logic next.

**Comment by jonathansampson** [comment]
After chatting with @tackley regarding the popup window (when you click the extension icon near the address bar), we've decided no action needs to be taken at the moment. It's unlikely this feature is used very often to begin with, and the best approach would be to do a complete overhaul of its current behavior. We'll revisit the popup functionality in a later PR.

**Review by tackley — APPROVED** [review]
Looks good, thanks!

Note I've just reviewed the code, not run against proton as I haven't got a proton calendar account to test against.

**Pull request merged**